go-controller/pkg/util/net_linux.go

```diff
@@ -167,3 +169,29 @@ func LinkNeighExists(link netlink.Link, neighIP net.IP, neighMAC net.HardwareAdd
 	}
 	return false, nil
 }
+
+func DeleteConntrack(ip string) {
+	ipAddress := net.ParseIP(ip)
+	if ipAddress == nil {
+		klog.Errorf("Value \"%s\" passed to DeleteConntrack is not an IP address", ipAddress)
+	}
+	filterTypes := []netlink.ConntrackFilterType{netlink.ConntrackOrigSrcIP, netlink.ConntrackOrigDstIP,
+		netlink.ConntrackReplySrcIP, netlink.ConntrackReplyDstIP, netlink.ConntrackReplyAnyIP}
```

Comment:
I think that netlink.ConntrackReplySrcIP, netlink.ConntrackReplyDstIP and netlink.ConntrackReplyAnyIP are redundant checking the implementation
https://github.com/vishvananda/netlink/blob/cf66001890388205fa46cba77a24a7586aae0c1c/conntrack_linux.go#L376-L389
```suggestion
	filterTypes := []netlink.ConntrackFilterType{netlink.ConntrackOrigSrcIP, 
	        netlink.ConntrackOrigDstIP,
		netlink.ConntrackReplyAnyIP}
```

Reply:
actually, I need @danwinship and @dcbw help here, 
but I think that we only need to delete those with the endpoint IP in nat `netlink.ConntrackReplyAnyIP`

Reply:
Yeah I think so too, just ReplyAnyIP should cover it.

Reply:
done